__tests__/app.test.js

```diff
@@ -69,4 +69,32 @@ describe("GET /api/articles/:article_id", () => (
             })
         })
     })
-));
\ No newline at end of file
+));
+
+describe("DELETE /api/comments/:comment_id", () => {
+    describe("Connecting to path", () => {
+        test("Should return status 204 for successful deletion", () => {
+            return request(app).delete("/api/comments/1").expect(204)
+        })
+        test("Should return no content for successful deletion", () => {
+            return request(app).delete("/api/comments/1")
+                .then(({ body }) => {
+                    expect(body).toEqual({})
+            })
```

Comment:
While I don't think this is an issue, a `204` implies no content already, so it may be unnecessary to test this